sdk/core/Azure.Core.Experimental/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 0.1.0-preview.16 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 0.1.0-preview.16 (2021-09-07)
```

Comment:
Did we want to change this prerelease label from preview to beta? I know this is a special package that will always remain in preview but was just curious if we thought about updating it.

Reply:
Didn't think much about it. We would have to go to 0.2.0-beta.1 but I think that's fine. I'll do it during the next release.